.gitignore

```diff
@@ -20,3 +20,4 @@ build/schema.graphql
 .config.env
 config.env
 jsconfig.json
+cypress.env.json
```

Comment:
it does seem like we'll need to add this config somewhere so that other devs can pull it onto their machine and so that the pipeline will have it when attempting to run the test.

Reply:
@mcstover where do you suggest it should be added at?

Reply:
It will live over in `marketplace-web-ui-ci` where you're adding it. we'll prob just want to add some documentation somewhere so folks know where to get it...